benchmarks/cloudsuite3/graph-analytics/run.sh

```diff
@@ -138,25 +132,28 @@ done
 rm -rf results
 mkdir -p results
 
-run_setup
+docker create --name graph-data cloudsuite3/twitter-dataset-graph
```

Comment:
Ideally, we would like to verify this command succeeds before proceeding and handle any errors appropriately.